core/allocators.h

```diff
@@ -61,60 +58,55 @@ class BalloonAllocator {
 
 public:
 	void *alloc(size_t p_size) {
-
 		size_t max = (1 << MAX_HANDS);
 		ERR_FAIL_COND_V(p_size > max, NULL);
 
 		unsigned int hand = 0;
-
 		while (p_size > (size_t)(1 << hand))
-			++hand;
+			hand++;
```

Comment:
No reason to change this

Reply:
Similar to above, what's more important:
- git blame noise
or
- code consistency (i.e. code noise).

Reply:
I prefer prefix over postfix. Since prefix cost more performance than postfix plus there is no need to change the code at all.